Core/FeatureFlag.swift

```diff
@@ -39,8 +39,10 @@ public enum FeatureFlag: String {
 extension FeatureFlag: FeatureFlagSourceProviding {
     public var source: FeatureFlagSource {
         switch self {
-        case .debugMenu, .sync, .appTrackingProtection:
+        case .debugMenu, .appTrackingProtection:
             return .internalOnly
+        case .sync:
+            return .remoteReleasable(.subfeature(SyncSubfeature.level0ShowSync))
```

Comment:
Shouldn't this be `.feature(.sync)` ?

Reply:
Yea, I think it is fine, as this subfeature is also guarded by the main feature state. Let's keep it. :)